admin/App.js

```diff
@@ -36,49 +37,44 @@ export class App extends Component {
     // Implementation
     //------------------
     renderNavBar() {
-        return navbar({
-            items: [
-                navbarGroup({
-                    align: 'left',
-                    items: [
-                        Icon.gears({size: '2x'}),
-                        navbarHeading(`${XH.appName} Admin`)
-                    ]
+        return appBar({
+            icon: Icon.gears({size: '2x'}),
+            title: `${XH.appName} Admin`,
+            leftItems: [
+                tabSwitcher({
+                    model: this.model.tabs,
+                    large: false
+                })
+            ],
+            rightItems: [
+                button({
+                    icon: Icon.mail(),
+                    text: 'Contact',
+                    onClick: this.onContactClick
+                }),
+                button({
+                    icon: Icon.openExternal(),
+                    title: 'Open app...',
+                    onClick: this.onOpenAppClick
                 }),
-                navbarGroup({
-                    align: 'right',
-                    items: [
-                        button({
-                            icon: Icon.mail(),
-                            text: 'Contact',
-                            onClick: this.onContactClick
-                        }),
-                        button({
-                            icon: Icon.openExternal(),
-                            title: 'Open app...',
-                            onClick: this.onOpenAppClick
-                        }),
-                        themeToggleButton(),
-                        logoutButton(),
-                        refreshButton({
-                            intent: 'success',
-                            onClick: this.onRefreshClick
-                        })
-                    ]
+                refreshButton({
+                    intent: 'success',
+                    onClick: this.onRefreshClick
                 })
-            ]
+            ],
+            adminButton: false
         });
     }
 
     onContactClick = () => {
         window.open('https://xh.io/contact');
-    }
+    };
```

Comment:
Have wanted to ask what we wanted as our standard here for semicolons after class properties. I think adding them is most consistent with our general standards. 

As per  https://github.com/tc39/proposal-class-public-fields/issues/25 it seems like it's generally more correct and avoids relying on ASI.

I took a quick look at our eslint rules and not entirely sure why it's not flagging them - https://github.com/exhi/hoist-dev-utils/blob/82d45641a1f9016c720e5e3a73a19322b4520e3f/packages/eslint-config/index.js#L83

Anyway - we should file a separate ticket to bomb the code and standardize if we go this way, and ideally get eslint to flag.